tests/sql/moz-fx-data-shared-prod/telemetry_derived/dataset_metadata.yaml

```diff
@@ -0,0 +1,13 @@
+friendly_name: User-Defined Functions
```

Comment:
nit: while these are for tests, mixing test metadata for `telemetry_derived` with what appears to have been metadata copied from `udf` is a bit confusing.